armory/scenarios/poisoning_witches_brew.py

```diff
@@ -419,6 +442,7 @@ def run_benign(self):
         self.probe.update(y=y, y_pred=y_pred)
 
         self.y_pred = y_pred  # for exporting when function returns
+        self.test_set_class_labels.update(y)
```

Comment:
is `self.test_set_class_labels` only supposed to be used if `hasattr(self, "fairness_metrics")`? Just confirming that's the case bc otherwise I don't see it being used

Reply:
Yes, fairness_metrics is the only part that uses `test_set_class_labels`